### PR TITLE
fix(ttx/npu): keep SwiGLU intermediate computations in FP32

### PR DESCRIPTION
**Pull request by shiwq-765**

### CODE REVIEW

**Review by shengw-bd — COMMENTED** [review]
LGTM

File: mojo_opset/tests/accuracy/operators/test_activation.py [hunk 1]
```diff
@@ -106,3 +106,26 @@ def test_rotate_activation(batch_size, seq_len, num_head, head_dim, dtype):
...
+    with bf16 inputs, that cost 2592593/9437184 forward elements, max|delta|
+    6.25e-2. With the fp32 intermediate the forward is bitwise equal to the fp32
+    reference, so this test asserts bitwise equality rather than a tolerance.
+    """
+    gate = torch.randn(*shape, dtype=torch.bfloat16)
+    up = torch.randn(*shape, dtype=torch.bfloat16)
+    swiglu = MojoSwiGLU()
+    actual = swiglu(gate, up)
+    expected = (torch.nn.functional.silu(gate.float()) * up.float()).to(gate.dtype)
+    torch.testing.assert_close(actual, expected, atol=0, rtol=0)
```
Comment (shengw-bd) [inline mojo_opset/tests/accuracy/operators/test_activation.py]: Is bitwise equality required here?

**Review by shengw-bd — APPROVED** [review]
